### PR TITLE
Use `latest` documentation version link

### PR DESCRIPTION
**Pull request by JackPGreen**

### CODE REVIEW

File: docs/modules/tutorials/pages/java-client-getting-started.adoc [hunk 1]
```diff
@@ -47,7 +47,7 @@ client.pfx
 client.truststore
 ----
 
-To understand and use the client, review the https://docs.hazelcast.com/hazelcast/5.3/clients/java#hide-nav[Java client documentation] to better understand what is possible.
+To understand and use the client, review the https://docs.hazelcast.com/hazelcast/latest/clients/java#hide-nav[Java client documentation] to better understand what is possible.
```
Comment (oliverhowell) [inline docs/modules/tutorials/pages/java-client-getting-started.adoc:50]:
```suggestion
To understand and use the client, review the https://docs.hazelcast.com/hazelcast/latest/clients/java[Java client documentation] to better understand what is possible.
```

File: docs/modules/tutorials/pages/java-client-getting-started.adoc [hunk 2]
```diff
@@ -427,7 +427,7 @@ In this tutorial, you learned how to get started with the Hazelcast Java Client,
 == See Also
 
 There are a lot of things that you can do with the Java Client. For more, such as how you can query a map with predicates and SQL,
-check out our https://github.com/hazelcast/hazelcast[Hazelcast repository] and our https://docs.hazelcast.com/hazelcast/5.3/clients/java#hide-nav[Java client documentation] to better understand what is possible.
+check out our https://github.com/hazelcast/hazelcast[Hazelcast repository] and our https://docs.hazelcast.com/hazelcast/latest/clients/java#hide-nav[Java client documentation] to better understand what is possible.
```
Comment (oliverhowell) [inline docs/modules/tutorials/pages/java-client-getting-started.adoc:430]:
```suggestion
check out our https://github.com/hazelcast/hazelcast[Hazelcast repository] and our https://docs.hazelcast.com/hazelcast/latest/clients/java[Java client documentation] to better understand what is possible.
```

**Review by oliverhowell — CHANGES_REQUESTED** [review]
LGTM but suggested removing the hide-nav links

**Comment by JackPGreen** [comment]
> LGTM but suggested removing the hide-nav links

Thanks but I'm not sure I understand the difference?

I've also realised that I've linked to the documentation using a "regular" link to the latest docs - but as were _in_ the documentation it probably should've been versioned (i.e. `5.3`'s docs should probably link to the `5.3` client page).

I think it'd have been better if I'd used the proper adoc linking syntax instead - do you know what that'd be, please?

File: docs/modules/tutorials/pages/java-client-getting-started.adoc (hunk 2 above)
Comment (oliverhowell) [reply docs/modules/tutorials/pages/java-client-getting-started.adoc]: > Thanks but I'm not sure I understand the difference?

The suggested change above ^ 
We don't need the #hide-nav on the end of the url - think it's an anachronism.

Comment (JackPGreen) [reply docs/modules/tutorials/pages/java-client-getting-started.adoc]: > The suggested change above ^ We don't need the #hide-nav on the end of the url - think it's an anachronism.

Sorry, I thought you were suggesting adding it, I didn't realise it was already there. That page has gone from upstream now anyway so a non-issue.

**Comment by oliverhowell** [comment]
> I think it'd have been better if I'd used the proper adoc linking syntax instead - do you know what that'd be, please?

You mean like this?

xref:{page-latest-supported-mc}@management-center:getting-started:overview.adoc[Management Center docs]

(from: https://github.com/hazelcast/hz-docs/blob/main/docs/modules/ROOT/pages/management-center.adoc)

**Comment by JackPGreen** [comment]
> > I think it'd have been better if I'd used the proper adoc linking syntax instead - do you know what that'd be, please?
> 
> You mean like this?
> 
> xref:{page-latest-supported-mc}@management-center:getting-started:overview.adoc[Management Center docs]
> 
> (from: https://github.com/hazelcast/hz-docs/blob/main/docs/modules/ROOT/pages/management-center.adoc)

Thanks, but I think it's a non-issue now. It was only the Java client docs that linked to another page within the same repo (`hz-docs`) but via the external URL.

File: docs/modules/maintain-cluster/pages/rolling-upgrades.adoc [hunk 3]
```diff
@@ -12,7 +12,7 @@ Before starting a rolling upgrade, consider the following best practices:
...
-* If using Hazelcast Management Center to trigger your rolling upgrade, upgrade Management Center before upgrading your members. For information on upgrading Management Center, refer to the link:https://docs.hazelcast.com/management-center/5.4/getting-started/install[Management Center documentation, window=_blank], or if you deployed Management Center using the Hazelcast Operator, refer to the link:https://docs.hazelcast.com/operator/latest/deploy-management-center.adoc[Operator documentation, window=_blank]
+* If using Hazelcast Management Center to trigger your rolling upgrade, upgrade Management Center before upgrading your members. For information on upgrading Management Center, refer to the link:https://docs.hazelcast.com/management-center/latest/getting-started/install[Management Center documentation, window=_blank], or if you deployed Management Center using the Hazelcast Operator, refer to the link:https://docs.hazelcast.com/operator/latest/deploy-management-center.adoc[Operator documentation, window=_blank]
```
Comment (oliverhowell) [inline docs/modules/maintain-cluster/pages/rolling-upgrades.adoc:15]:
```suggestion
* If using Hazelcast Management Center to trigger your rolling upgrade, upgrade Management Center before upgrading your members. For information on upgrading Management Center, refer to the xref:{page-latest-supported-mc}@management-center:getting-started:install.adoc[Management Center documentation], or if you deployed Management Center using the Hazelcast Operator, refer to the https://docs.hazelcast.com/operator/latest/deploy-management-center[Hazelcast Platform Operator documentation].
```

Comment (oliverhowell) [reply docs/modules/maintain-cluster/pages/rolling-upgrades.adoc:15]: Check links work but I would suggest not using the external link syntax for our own docs, especially with the new product selector on every page.

**Review by oliverhowell — APPROVED** [review]

**Pull request merged**